services/hexathons/src/permission.ts

```diff
@@ -28,6 +31,9 @@ export const addAbilities = (): RequestHandler => (req, res, next) => {
 
   can("read", "Hexathon", { isActive: true });
   can("read", "Interaction", { userId: req.user.uid });
+  can("read", "Event", { userId: req.user.uid });
+  can("read", "Location", { userId: req.user.uid });
+  can("read", "Tag", { userId: req.user.uid });
```

Comment:
For this, you don't need the filter as they're aren't individual to a user, so you can just keep it for example `can("read", "Event")`